telegram/ext/_applicationbuilder.py

```diff
@@ -543,6 +559,24 @@ def pool_timeout(self: BuilderType, pool_timeout: Optional[float]) -> BuilderTyp
         self._pool_timeout = pool_timeout
         return self
 
+    def http_version(self: BuilderType, http_version: str) -> BuilderType:
+        """Sets the HTTP protocol version which is used for the
+        :paramref:`~telegram.request.HTTPXRequest.http_version` parameter of
+        :attr:`telegram.Bot.request`. By default, HTTP/2 is used.
+
+        .. versionadded:: 20.1
+
+        Args:
+            http_version (:obj:`str`): Pass `1.1` if you'd like to use HTTP/1.1 for making requests
```

Comment:
```suggestion
            http_version (:obj:`str`): Pass ``"1.1"`` if you'd like to use HTTP/1.1 for making requests
```

same in other places :)